x-pack/plugins/security_solution/public/onboarding/components/onboarding_body/cards/assistant/assistant_check_complete.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { loadAllActions as loadConnectors } from '@kbn/triggers-actions-ui-plugin/public/common/constants';
+import type { AIConnector } from '@kbn/elastic-assistant/impl/connectorland/connector_selector';
+import type { OpenAiProviderType } from '@kbn/stack-connectors-plugin/public/common';
+import type { OnboardingCardCheckComplete } from '../../../../types';
+import { AllowedActionTypeIds } from './constants';
+
+export const checkAssistantCardComplete: OnboardingCardCheckComplete = async ({ http }) => {
+  const aiConnectorsResult = await loadConnectors({ http });
```

Comment:
These are not _aiConnectors_ yet, this function loads all the connector types (including non-AI), the filtering we do later using `AllowedActionTypeIds` is what make them aiConnectors. 
Can we rename the variable and use the original function name?
```suggestion
  const allConnectors = await loadAllActions({ http });
```